plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/listeners/CodeWhispererCodeScanEditorMouseMotionListener.kt

```diff
@@ -193,7 +211,27 @@ class CodeWhispererCodeScanEditorMouseMotionListener(private val project: Projec
             isEditable = false
             addHyperlinkListener { he ->
                 if (he.eventType == HyperlinkEvent.EventType.ACTIVATED) {
-                    BrowserUtil.browse(he.url)
+                    when {
+                        he.description.startsWith("amazonq://issue/explain-") -> {
+                            val issueItemMap = mutableMapOf<String, String>()
```

Comment:
we have a type system. Use a `data class` instead of throwing everything into an untyped map

Reply:
Let me add that in followup commit.